### PR TITLE
HW-51706 Support Editability enabled/disabled (UI)

### PR DESCRIPTION
**Pull request by ymoskovko-epam**

### CODE REVIEW

File: Sources/Helper/Theme.swift [hunk 1]
```diff
@@ -72,6 +72,8 @@ public struct Theme {
     public struct Text {
         /// The text primary color
         public static var color = UIColor.black
+        /// The text disabled color
+        public static var disabledColor = UIColor(rgb: 0x8e8e93)
```
Comment (jsaini-hw) [inline Sources/Helper/Theme.swift]: There shouldn't be disabled color. It should be same as label color

File: Sources/View/Widget/AbstractWidget.swift [hunk 2]
```diff
@@ -23,6 +23,10 @@ import UIKit
 class AbstractWidget: UIStackView, UITextFieldDelegate {
     var field: HyperwalletField!
 
+    var isEditable: Bool {
```
Comment (jsaini-hw) [inline Sources/View/Widget/AbstractWidget.swift]: This should go inside SelectionWidget.

Comment (ymoskovko-epam) [reply Sources/View/Widget/AbstractWidget.swift]: it is used in TextWidget as well.

Comment (jsaini-hw) [reply Sources/View/Widget/AbstractWidget.swift]: TextWidget should directly use isEditable from field. TextWidget has Field. Only for selection widget add that logic inside SelectionWidget

File: Sources/Helper/Theme.swift (hunk 1 above)
Comment (ymoskovko-epam) [reply Sources/Helper/Theme.swift]: fixed

File: Sources/View/Widget/AbstractWidget.swift (hunk 2 above)
Comment (ymoskovko-epam) [reply Sources/View/Widget/AbstractWidget.swift]: `Field` property is in AbstractWidget and all descendant widgets have access to it. Why we should do copy/paste logic?

Comment (jsaini-hw) [reply Sources/View/Widget/AbstractWidget.swift]: That's the point Field property is in AbstractWidget and all the descendant widgets have access to it so why create another variable for every cell when we can directly check isEditable from field in the descendant widgets?

File: Sources/TransferMethod/AddTransferMethodViewController.swift [hunk 3]
```diff
@@ -218,7 +218,7 @@ extension AddTransferMethodViewController {
         }
         let widget = presenter.sections[indexPath.section][indexPath.row]
         cell.contentView.addSubview(widget)
-        if let widget = widget as? SelectionWidget {
+        if let widget = widget as? SelectionWidget, widget.isEditable {
```
Comment (jsaini-hw) [inline Sources/TransferMethod/AddTransferMethodViewController.swift]: this can be widget.field.isEditable

Comment (ymoskovko-epam) [reply Sources/TransferMethod/AddTransferMethodViewController.swift]: ok

**Comment by ymoskovko-epam** [comment]
@jsaini-hw looks good. thank you for help.

**Pull request merged**